
  float32 brake
  float32 throttle
  float32 steering
  bool steering_mult
  int16 gear_cmd
  int16 turn_signal_cmd
